README.md

```diff
@@ -162,6 +162,17 @@ laton -g main.tex
 
 The script will behave as if `main.tex` includes each file stored in your git repo.
 
+### Example 3: compile remote git repo (which you have checked out locally)
```

Comment:
```suggestion
### Example 4: compile remote git repo (which you have checked out locally)
```

Also, an interesting workflow! It feels inconvenient to have to push things to git first before
compiling it locally. Why doesn't the regular `laton -g main.text` work for you?

Reply:
Well, I personally do not use `latexonline.cc` but my own version, which actually has an ssh key and pass me the pub key so I could add it as deploy key. 

Well sure the workflow with the tar is ok, but why send data throughout the net many times, if my instance runs on the same server as my git(ea).